apps/explorer/src/hooks/useNetwork.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
This is somewhat copy + pasted from `contexts.ts`, but I cleaned it up and added better typing